aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,49 @@
 module aptos_framework::transaction_context {
+
+    use std::features;
+
+    /// AUID feature is not supported.
+    const EAUID_NOT_SUPPORTED: u64 = 3;
+
+    /// A wrapper denoting aptos unique identifer (AUID)
+    /// for storing an address
+    struct AUID has drop, store {
```

Comment:
Should this be copyable as well?

Reply:
GUID correctly has only drop, store. So, I used the capabilities here. I added copy now.

Reply:
It shouldn't be copyable.

Main point is if you have two instances of AUID, they are guaranteed to be different. 
copyable breaks that